quodlibet/quodlibet/qltk/songlistcolumns.py

```diff
@@ -279,14 +279,23 @@ def _apply_value(self, model, iter_, cell, stamp):
             except (OverflowError, ValueError, OSError):
                 text = u""
             else:
-                today = datetime.datetime.now().date()
-                days = (today - date).days
-                if days == 0:
-                    format_ = "%X"
-                elif days < 7:
-                    format_ = "%A"
+                format_setting = config.gettext("settings",
+                                      "datecolumn_timestamp_format")
+
+                # use default behaviour-format
+                if not format_setting:
```

Comment:
Why not swap the order of the if-else and avoid the `not` (always harder to read)?